server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/message/MessageBuilder.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaaproject.kaa.server.transport.message;
+
+/**
+ * Converts message data into objects specific to corresponding transport
```

Comment:
Converts the message data into objects specific to the corresponding transport channel.